src/main/java/programminglife/model/DataManager.java

```diff
@@ -237,11 +246,13 @@ public static void clearDB(String name) throws IOException {
      * completely remove a database. This cannot be undone.
      * By default, this method removes first converts the name with {@link #toDBFile(String)}
      * @param name The name of the database to be removed.
+     * @return true if the file was removed, false if it did not exist
      * @throws IOException When something goes wrong with IO
      */
-    public static void removeDB(String name) throws IOException {
+    public static boolean removeDB(String name) throws IOException {
+        System.out.printf("[%s] Removing database %s%n", Thread.currentThread().getName(), name);
         close();
-        Files.deleteIfExists(new File(DataManager.toDBFile(name)).toPath());
+        return Files.deleteIfExists(Paths.get(new File(DataManager.toDBFile(name)).getName()));
```

Comment:
what is the difference between `Paths.get([File])` and `[File].getPath()` (and why is `Paths.get` better)?